apps/system/js/notifications.js

```diff
@@ -624,18 +657,18 @@ var NotificationScreen = {
     notification.style.transform = '';
   },
 
-  addUnreadNotification: function ns_addUnreadNotification(id, skipUpdate) {
+  addUnreadNotification: function ns_addUnreadNotification(appManifestURL, skipUpdate) {
     if (UtilityTray.shown) {
       return;
     }
-    this.unreadNotifications.push(id);
+    this.unreadNotifications.push({name: appManifestURL, count: 1});
```

Comment:
better to use 'appId' or something similar for the first attribute name instead of 'name' used now.